uber/models.py

```diff
@@ -630,9 +630,22 @@ def everything(self, location=None):
                                         .order_by(Attendee.full_name).all()
                          if c.AT_THE_CON or not location or int(location) in a.assigned_depts_ints]
             for job in jobs:
-                job._available_staffers = [a for a in attendees if not job.restricted or a.trusted]
+                job._available_staffers = [a for a in attendees if not job.restricted or a.trusted_in(job.location)]
             return jobs, shifts, attendees
 
```

Comment:
OH. wild.  that is some voodoo magic right there.
though, would that mean that if Job.available_staff is a cached property, the only time it's going to get updated is when someone calls session.everything() ?  could that result in stale cache data?

I'm pretty sure I hit this myself during testing with some bewildering cached data that magically went away when I restarted the server (didn't realized it was a cached_property til later).

Should we still keep it this way?  I'm sure this was put here to work around the original implementation being really slow on a large production DB.  If you'd like I could profile it with a large dataset before and after, and if the new query is way faster [which, it really should be...] (famous last words), then we might not need any of this anymore.